Core/include/Acts/EventData/TrackStateType.hpp

```diff
@@ -115,8 +115,8 @@ class ConstTrackStateType {
     for (std::size_t i = 0; i < TrackStateFlag::NumTrackStateFlags; i++) {
       trunc[i] = bs[i];
     }
-    // MeasurementParameterOutlierHoleMaterialSharedhit
-    os << "MPOHMS=" << trunc;
+    // SharedhitMaterialHoleOutlierParameterMeasurement
```

Comment:
Ok I see.
Should this print the literal byte content, or should it print the ordering in the enum though?

Reply:
I'd be happy to do it the other way. This way seemed the simplest unless someone had an opinion one way or the other.

Reply:
I have no opinion on the order as long as it is consistent 😄 anyways if somebody sees a problem we can always change it

Reply:
Let's merge it as is.